packages/protocol/src/action/protocols.ts

```diff
@@ -108,10 +122,20 @@ export interface ClusterTypeProtocol extends AddressableElementProtocol<ClusterI
      */
     attributes: CollectionProtocol<AttributeTypeProtocol>;
 
+    /**
+     * Map of attribute names to IDs.
+     */
+    attributeNameToId: Map<string, AttributeId>;
```

Comment:
Is this used anywhere? If we need it we could add the name to the ClusterTypeProtocol

Reply:
Yes used because the state only has the property names that were changed, so we convert that in protocolService to be ids rather than numbers. name is already in clustertype protocol - but not fast to access

Reply:
attributeNameToId stays because used by ProtocolService - and likely too for the change event adjustment events

Reply:
ProtocolService's requirements shouldn't affect the ClusterProtocol... That should only contain fields required by @matter/orotocol...  Which should only be dealing with IDs except for diagnostics, right?

Reply:
I found a solution that should work

Reply:
and also can remove that now :-) I just filter on a different place